### ISSUE TITLE
Errors with rosetta refine

### ISSUE THREAD

**Issue by ryanmcgreevy** [ryanmcgreevy/ModelMaker#11]
A new [commit](https://github.com/ryanmcgreevy/ModelMaker/commit/b19348557dcf9088a910aacbcf64d33eb58b3ec9) from me includes an initial wrapper for start_rosetta_refine, without any modification to the underlying package code. However, I get errors when trying to run the tutorial example. This error happens even when I run it from the script provided in the tutorial. Depending on whether I use Rosetta 3.7 or 3.8, I get a different error. The outputs for each are below.

**Rosetta 3.8:**
```
From line 3:
Error: Element 'cen': This element is not expected. Expected is ( ScoreFunction ).

1: <ROSETTASCRIPTS>
2: <SCOREFXNS>
3:    <cen weights="score4_smooth_cart">
4:      <Reweight scoretype="elec_dens_fast weight=20"/>
5:    </cen>
6:    <dens_soft weights="soft_rep">
7:      <Reweight scoretype="cart_bonded" weight="0.5"/>
8:      <Reweight scoretype="pro_close" weight="0.0"/>
From line 36:
Error: Element 'Span': This element is not expected. Expected is one of ( span, Seeds, residue ).

31: <Span begin="274" end="284" chi="1" bb="1" />
32: </MoveMap>
33: </FastRelax>
34: <CartesianSampler name="cen5_50" automode_scorecut="-0.3" scorefxn="cen" mcscorefxn="cen" fascorefxn="dens_soft" strategy="auto" fragbias="density" rms="1.5" ncycles="200" fullatom="0" bbmove="0" nminsteps="25" temp="4" fraglens="7" nfrags="25" residues_to_exclude="1-194,202-278" />
35: <CoordinateCst name="coordcst" stddev="20" atom="CA" anchor_res="1" jump="0">
36: <Span begin="1" end="189" />
37: <Span begin="207" end="273" />
38: </CoordinateCst>
39: </MOVERS>
40: <PROTOCOLS>
41: <Add mover="setupdens"/>
```

**Rosetta 3.7:**
```
Error: ERROR: Exception caught by JobDistributor while trying to get pose from job 'rpn11_midres_bb_rpn11_yeast_0001'
490 Error: Option weight not found.
```

**Comment by maxscheurer** [comment]
Probably, I tried this with an old Rosetta version, but with 3.8, I get the same output to the my logfile.

I figured out that we need to change the code to support the new Rosetta 3.8 XML conventions.

Using the `rewrite_rosetta_script.py` script in `tools/xsd_xrw`, we can in a way determine
what needs to be changed.
However, I just noticed that some options are not converted properly. So we need to
figure out from the non-existing documentation how to convert stuff... This will be fun. :fire:
